app/javascript/articles/components/__tests__/ReactionsCount.test.jsx

```diff
@@ -0,0 +1,28 @@
+import { h } from 'preact';
+import { render } from '@testing-library/preact';
+import '@testing-library/jest-dom';
+import { ReactionsCount } from '..';
+import {
+  articleWithReactions,
+  articleWithoutReactions,
+} from '../../__tests__/utilities/articleUtilities.js';
+import '../../../../assets/javascripts/lib/xss';
+import '../../../../assets/javascripts/utilities/timeAgo';
```

Comment:
are these needed?

Reply:
😅  you're right. I copied my test from the `Article.test.jsx`; these imports are the remnants of that move. I shall delete.